cmake/DD4hepConfig.cmake.in

```diff
@@ -60,18 +60,18 @@ if(DD4HEP_USE_GEANT4)
   DD4HEP_SETUP_GEANT4_TARGETS()
 endif()
 
-FIND_DEPENDENCY(ROOT REQUIRED)
+FIND_DEPENDENCY(ROOT REQUIRED CONFIG)
 DD4HEP_SETUP_ROOT_TARGETS()
 
 if(DD4HEP_USE_LCIO)
   MESSAGE(STATUS "@CMAKE_PROJECT_NAME@ uses LCIO")
-  find_dependency(LCIO REQUIRED)
+  find_dependency(LCIO REQUIRED CONFIG)
   DD4HEP_SETUP_LCIO_TARGETS()
 endif()
 
 if(DD4HEP_USE_TBB)
   MESSAGE(STATUS "@CMAKE_PROJECT_NAME@ uses TBB")
-  find_dependency(TBB REQUIRED)
```

Comment:
what bout adding this also in the main cmakelists?

Reply:
Yes, done.